src/main/java/com/weaverplatform/importer/xmi/ImportXmi.java

```diff
@@ -149,20 +218,22 @@ private String getValidSubNodeValue(XML association) {
 
   /**
    * Save the name of an xmi-class as weaver individual
+   *
    * @param xmiClasses
    */
   public void createWeaverIndividuals(HashMap<String, String> xmiClasses) {
     Iterator it = xmiClasses.entrySet().iterator();
     while (it.hasNext()) {
-      Map.Entry pair = (Map.Entry)it.next();
-      String xmiClassName = (String)pair.getValue();
+      Map.Entry pair = (Map.Entry) it.next();
+      String xmiClassName = (String) pair.getValue();
       //save to weaver as Individual
```

Comment:
Please use generics here, instead of casts.

Iterator<Map.Entry<String, String>> it
Map.Entry<String, String> pair

Either that or just iterate over xmlClasses.values() and get rid of all the intermediate variables.